lib/Sema/DerivedConformanceAdditiveArithmeticVectorNumeric.cpp

```diff
@@ -78,67 +78,103 @@ static ValueDecl *getProtocolRequirement(ProtocolDecl *proto, Identifier name) {
 
 // Return the `Scalar` associated type for a ValueDecl if it conforms to
 // `VectorNumeric` in the given context.
-// If the decl does not conform to `VectorNumeric`, return a null `Type`.
+// If the decl does not conform to `VectorNumeric`, return nullptr.
 static Type getVectorNumericScalarAssocType(VarDecl *decl, DeclContext *DC) {
   auto &C = decl->getASTContext();
   auto *vectorNumericProto = C.getProtocol(KnownProtocolKind::VectorNumeric);
   if (!decl->hasType())
     C.getLazyResolver()->resolveDeclSignature(decl);
   if (!decl->hasType())
-    return Type();
+    return nullptr;
   auto declType = decl->getType()->hasArchetype()
                       ? decl->getType()
                       : DC->mapTypeIntoContext(decl->getType());
   auto conf = TypeChecker::conformsToProtocol(declType, vectorNumericProto, DC,
                                               ConformanceCheckFlags::Used);
   if (!conf)
-    return Type();
+    return nullptr;
   Type scalarType = ProtocolConformanceRef::getTypeWitnessByName(
       declType, *conf, C.Id_Scalar, C.getLazyResolver());
   assert(scalarType && "'Scalar' associated type not found");
   return scalarType;
 }
 
-static Type deriveVectorNumeric_Scalar(NominalTypeDecl *nominal,
+// Returns the `Scalar` associated type for a nominal type with the given
```

Comment:
Other places used bare form.
```suggestion
// Return the `Scalar` associated type for a nominal type with the given
```

Reply:
The predominant verbiage seems to have been `Returns`, but I also prefer `Return`.
Edited in https://github.com/apple/swift/pull/22114/commits/c6ef5a46690970392947c3751cc6cd142277b158.